docs/src/embedding.md

```diff
@@ -0,0 +1,90 @@
+# Embedding Functions
+
+Embeddings are high dimensional floating-point vector representations of your data or query.
+Anything can be embedded using some embedding model or function.
+For a given embedding function, the output will always have the same number of dimensions.
+
+## Creating an embedding function
+
+Any function that takes as input a batch (list) of data and outputs a batch (list) of embeddings
```

Comment:
is input always the same length as outputs?

Reply:
it should. i'll add a note